style.css

```diff
@@ -0,0 +1,102 @@
+.nav__link {
+  cursor: pointer;
+  color: black;
+  padding: 15px 10px;
+  text-decoration: none;
+  text-align: center;
+  font-size: 36px;
+}
+
+.burger {
+  position: relative;
+  display: none;
+  left: 0;
+  top: 5px;
+  content: '';
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  background: black;
+  opacity: 50%;
+}
+
+.burger::before {
+  content: '';
+  display: block;
+  width: 20px;
+  height: 2px;
+  background: #fff;
+  position: absolute;
+  left: 2px;
+  }
+
+.burger::after {
+  content: '';
+  display: block;
+  width: 20px;
+  height: 2px;
+  background: #fff;
+  position: absolute;
+  left: 2px;
+  }
+
+.content { font-size: 36px; }
+ 
+@media screen and (max-width: 480px) {
+	
+  .nav__link { display: none; }
+
+  .burger { display: block; }
+
+  .burger::before {
+   position: relative;
+   left: 5px;
+   top: 9.25px;
+   content: '';
+   width: 15px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Azez1/pullRequest?prid=5948917)